stanford/bin/cut-release-branch

```diff
@@ -13,7 +13,9 @@ done
 : ${BRANCH_TO_CUT:=rc}
 : ${WORKSPACE:=${HOME}/workspace/cut_branch}
 : ${EMAIL_RECIPIENT:=''}
-date=$(echo "${BUILD_ID}" | head -c10)
+: ${EMAIL_SENDER:= 'no_reply@lagunita.stanford.edu'}
+: ${EMAIL_SENDER_NAME:= 'R C Cutter'}
```

Comment:
I think in the past we've just used `jenkins`,
which makes this easier to search.

Reply:
And same note here about superfluous space.